website/blog/2019-11-04-binding-react-navigation-5-to-ui-kitten.md

```diff
@@ -0,0 +1,429 @@
+
+<img src="assets/binding-react-navigation-5-to-ui-kitten/overview.gif" height="480"/>
+
+## Overview
+
+React Navigation 5 is nothing else rather than simplifying navigation structure in your app.
+
+```js
+import { createStackNavigator } from '@react-navigation/stack';
+
+const Stack = createStackNavigator();
+
+export const AuthNavigator = (): React.ReactElement => (
+  <Stack.Navigator headerMode='none'>
+    <Stack.Screen name='Sign In' component={SignInScreen}/>
+    <Stack.Screen name='Sign Up' component={SignUpScreen}/>
+  </Stack.Navigator>
+);
+```
+
+The syntax is simple. You just import `createXNavigator` from the navigator package of your choice and use `Navigator` and `Screen` components from the value returned by this function.
+
+Unlike the previous React Navigation versions, all the screens used inside a navigator are passed as child elements with wrapping it to a `Screen` component. If you need to set up additional navigator configuration like configuration of the header, you can simply pass corresponding props directly to the `Navigator` component.
+
+## Getting started
+
+Clone the project from GitHub. It contains all the required source code for the initial setup.
+
+```bash
+git clone https://github.com/artyorsh/react-navigation-ex-demo
+```
+
+### Step 1. Authentication flow
+
+Assuming that your app users will need to authorize before getting to the home screen, we will need to create both Authentication and Home navigators. Then we're going to combine it with simple stack navigation and pick the initial screen depending on the user authorization status.
+
+Open `./src/navigators/auth.navigator.tsx` file and paste the following code:
+
+
+```js
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import { SignInScreen, SignUpScreen, ResetPasswordScreen } from '@app-scenes/auth';
+import { AppRoute } from './app-routes';
+
+const Stack = createStackNavigator();
+
+export const AuthNavigator = (): React.ReactElement => (
+  <Stack.Navigator headerMode='none'>
+    <Stack.Screen name={AppRoute.SIGN_IN} component={SignInScreen}/>
+    <Stack.Screen name={AppRoute.SIGN_UP} component={SignUpScreen}/>
+    <Stack.Screen name={AppRoute.RESET_PASSWORD} component={ResetPasswordScreen}/>
+  </Stack.Navigator>
+);
+```
+
+In this example, we're using a `createStackNavigator` function to create simple stack navigation between Sign In, Sign Up and Reset Password screens. Under `Stack Navigator` we mean the default navigation behavior between screens: with slide-from-right animation on iOS, and slide-in-top on Android.
+
+In app.navigator.tsx file and replace the placeholder screen with Auth Navigator. This will make authentication screens to be the starter point of your app.
```

Comment:
```suggestion
In `app.navigator.tsx` file and replace the placeholder screen with Auth Navigator. This will make authentication screens to be the starter point of your app.
```